books/GNUmakefile

```diff
@@ -413,9 +413,9 @@ $(info Excluding books that need STP: [$(CERT_PL_USES_STP)])
 OK_CERTS := $(filter-out $(CERT_PL_USES_STP), $(OK_CERTS))
 endif
 
-ifeq ($(OS_HAS_CPP), )
-$(info Excluding books that invoke cpp: [$(CERT_PL_USES_CPP)])
-OK_CERTS := $(filter-out $(CERT_PL_USES_CPP), $(OK_CERTS))
+ifeq ($(OS_HAS_GCC_C17), )
+$(info Excluding books that invoke GCC with \"-std=c17\": [$(CERT_PL_USES_GCC_C17)])
```

Comment:
This isn't incorrect, but it struck me as suggesting that some books invoke GCC without that flag. I think it would be a little bit clearer to phrase it as "Excluding books that require a GCC with \"-std=c17\""